### PR TITLE
incorporated comments for module 5 notebook

### PR DESCRIPTION
**Pull request by mzraghib**
*Issue #, if available:*

*Description of changes:*
Incorporated comments from pull request: 
1)Are you able to link the blogpost here? 
Blog link is not release yet, will update when it is published.
2)Consistently with other modules, can you please include a step to install relevant libraries.
Added

3)can you keep the python code consistent, using snake_case rather than PascalCase.
Used snake_case for consistency
   
 4) Fixed typo -->   columnds -> columns

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by matteofigus — APPROVED** [review]

File: identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac140c42",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import json\n",
```
Comment (ctd) [inline identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb]:
```suggestion
```

This library doesn't appear to be in any of the following code.

File: identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    "import io\n",
+    "import pandas as pd\n",
+    "from io import BytesIO\n",
```
Comment (ctd) [inline identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb]:
```suggestion
```

This library doesn't appear to be in any of the following code.

**Review by ctd — APPROVED** [review]
Two small things, otherwise LGTM.

File: identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb (hunk 2 above)
Comment (matteofigus) [reply identity-verification/module_5/Metrics_for_Evaluating_an_Identity_Verification_Solution.ipynb:76]: This is used in line 122

**Pull request merged**